src/firmware/cmake_binary_builders/build_firmware.cmake

```diff
@@ -21,21 +21,22 @@ function(build_firmware FIRMWARE_SOURCE_DIR SRC_FILES BINARY_NAME OUT_DIR)
 
     # a list of the c flags we are going to use
     set(CFLAGS
-        -std=gnu99
-        -O2
-        -mfloat-abi=hard
-        -mlittle-endian
-        -mcpu=cortex-m4
-        -mfpu=fpv4-sp-d16
-        -mthumb
-        -ggdb3
-        -fno-common
-        -ffunction-sections
-        -static
-        -Wall
-        -Wextra
-        -Wdouble-promotion
-        -Wpointer-arith)
+"        -std=gnu99",
```

Comment:
Do we still need cmake? Is bazel using cmake to build somehow?

Reply:
Whoops, this file should not have been changed, reverted. We do still need CMake for some things like building code for the dongle. Will remove after transition to bazel is complete.